infra/dev/laravel/Dockerfile

```diff
@@ -0,0 +1,43 @@
+# use Debian Buster (10) with FPM
```

Comment:
Remove all the commented stuff, it clutters the file

Reply:
Fixed in 4ab3fd5